packages/components/src/components/types/icons.types.ts

```diff
@@ -0,0 +1,30 @@
+export interface IIconsManifest {
+    // If a new icon category is added, be sure to add it here as well. This is part of the fix for ts(7053)
+    readonly brand: string;
+    readonly cashier: string;
+    readonly common: string;
+    readonly contract: string;
+    readonly currency: string;
+    readonly dxtrade: string;
+    readonly flag: string;
+    readonly mt5: string;
+    readonly option: string;
+    readonly stock: string;
+    readonly tradetype: string;
+    readonly underlying: string;
+    readonly wallet: string;
+}
+export type TIconProps = {
+    className?: string;
+    color?: string;
```

Comment:
would you please check this one too? I see that the previous type also included `boolean`

Reply:
@iman-fs, based on it's usage, `color` will always be a string within this component
<img width="676" alt="color" src="https://user-images.githubusercontent.com/104334373/189068193-10191095-4595-45ef-bd55-a2958674eba0.png">

Reply:
ok then. thanks.